tests/test_valuerep.py

```diff
@@ -0,0 +1,36 @@
+import pytest
```

Comment:
@CPBridge I assume we no longer need these tests in highdicom, do we?

Reply:
No this PR was originally written with the assumption that the functionality would be in highdicom and I haven't adapted it to at all yet for the fact that the functionality has moved to pydicom. A lot will need to change. I might just create a new PR from scratch, not sure which will end up being easier